### ISSUE TITLE
Markdown-toc fails if there are multiple .md files

### ISSUE THREAD

**Issue by nymous** [Lucas-C/pre-commit-hooks-nodejs#8]
After #4 landed the precommit fails if multiple Markdown files are present, because `markdown-toc` doesn't support multiple input files on its CLI. This is sad because it was the point of that PR :sweat_smile:

This issue has been reported by Yajo itself here https://github.com/pre-commit/pre-commit/issues/1374

My current workaround is to use the pre-commit offered by [DocToc](https://github.com/thlorenz/doctoc#usage-as-a-git-hook) (as mentioned in the pre-commit issue).

Note: This issue is more of a heads-up for others who would run into the same problem because it would need to be fixed in the `markdown-toc` project directly.

**Comment by Lucas-C** [comment]
I opened https://github.com/jonschlinkert/markdown-toc/issues/162 to suggest the evolution in `markdown-toc`

**Comment by nymous** [comment]
Awesome! I was checking in the upstream issues to see if someone already asked for this before creating a new one but you beat me to it ^^

**Comment by beliaev-maksim** [comment]
would be helpful to have it

now have to use multiple hooks for each file:
~~~yaml
  - repo: https://github.com/Lucas-C/pre-commit-hooks-nodejs
    rev: v1.1.1
    hooks:
      - id: markdown-toc
        name: README.md-toc
        files: ^README\.md$
      - id: markdown-toc
        name: configuration.md-toc
        files: .*/configuration\.md$
      - id: markdown-toc
        name: CONTRIBUTE.md-toc
        files: .*/CONTRIBUTE\.md$
~~~

**Comment by Lucas-C** [comment]
This depend on https://github.com/jonschlinkert/markdown-toc/issues/162
You may want to drop a comment there

**Comment by beliaev-maksim** [comment]
@Lucas-C 
we got a green light for update of CLI

would you like to create a PR for this?

**Comment by drjasonharrison-vp-eio** [comment]
What do we need to fix this?

**Comment by Lucas-C** [comment]
Have this PR merged: https://github.com/jonschlinkert/markdown-toc/pull/172

**Comment by drjasonharrison** [comment]
How much longer to wait until we just switch to your fork of the repo?
Or maybe @Lucas-C would be willing to take on the latest leading fork? https://github.com/Lucas-C/markdown-toc

**Comment by beliaev-maksim** [comment]
+1 on switching the fork until they merge

**Comment by Lucas-C** [comment]
I released a `v1.1.2` version that uses [my GitHub fork of the `markdown-toc` lib](https://github.com/Lucas-C/pre-commit-hooks-nodejs/blob/v1.1.2/.pre-commit-hooks.yaml#L25)

This issue should be solved now

**Comment by drjasonharrison** [comment]
Thanks @Lucas-C!
